src/ToggleButton/ToggleButton.stories.tsx

```diff
@@ -0,0 +1,783 @@
+/* ======================================================================== *
+ * Copyright 2024 HCL America Inc.                                          *
+ * Licensed under the Apache License, Version 2.0 (the "License");          *
+ * you may not use this file except in compliance with the License.         *
+ * You may obtain a copy of the License at                                  *
+ *                                                                          *
+ * http://www.apache.org/licenses/LICENSE-2.0                               *
+ *                                                                          *
+ * Unless required by applicable law or agreed to in writing, software      *
+ * distributed under the License is distributed on an "AS IS" BASIS,        *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. *
+ * See the License for the specific language governing permissions and      *
+ * limitations under the License.                                           *
+ * ======================================================================== */
+
+import React from 'react';
+import { StoryFn, Meta } from '@storybook/react';
+import IconAdd from '@hcl-software/enchanted-icons/dist/carbon/es/add';
+import { Grid } from '@mui/material';
+
+import ToggleButton, { ToggleButtonSizes, ToggleButtonVariants } from './ToggleButton';
+import Typography from '../Typography';
+import Divider from '../Divider';
+
+export default {
+  title: 'Inputs/ToggleButton',
+  component: ToggleButton,
+  argTypes: {
```

Comment:
Please checkout the ToggleButton storybook page, there are some controls that we don't encourage Storybook users to touch or needs more work if they want to do it.

For other components, we've made these controls readonly or totally removed from table https://storybook.js.org/docs/essentials/controls#disable-controls-for-specific-properties. A good guide to follow would be the IconButton since they're kinda close.